db/migrate/20170410164012_remove_workspaces_collection_foreign_key.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveWorkspacesCollectionForeignKey < ActiveRecord::Migration[5.0]
+  def change
+    remove_foreign_key :workspaces, :collection
```

Comment:
The migration doesn't complain, but I don't think we'll know for sure until we try to run this migration under a database that supports foreign key constraints.